docs/09-troubleshooting/common-issues.mdx

```diff
@@ -2,8 +2,93 @@
 
 Below you'll find a collection of answers to commonly reported problems.
 
+## Unity Activation
+
+If you are having troubles with activation, you are not alone, this is a common problem and it is
+not the easiest thing to solve. Here are some tips:
+
+- Try removing special characters from your password, sometimes they cause problems with command
+  lines. _This is not a security advice_
+- Your username should be an email
+- Confirm you are able to connect on Unity's website using your credentials
+
+:::caution Known issue with gitlab-ci activation and empty license file
```

Comment:
We will want to remove that once this is solved on the gitlab side, but this should help us a bit by pointing people in the right direction until this gets fixed. There has been a few people asking for this already on discord.